bin/lint-staged.js

```diff
@@ -29,7 +31,7 @@ const debug = debugLib('lint-staged:bin')
 cmdline
   .version(pkg.version)
   .option('--allow-empty', 'allow empty commits when tasks revert all staged changes', false)
-  .option('-c, --config [path]', 'path to configuration file')
+  .option('-c, --config [path]', 'path to configuration file, or - to read from stdin')
```

Comment:
We should add this to the readme, since I don't clearly see how it works.

Is it like this:

```bash
npx lint-staged --config - <some json here>
```

Reply:
Usage would be `./generate-lint-staged-config.py | npx lint-staged --config -`

I'll add to `README.md`